open-api/rest-catalog-open-api.yaml

```diff
@@ -1630,6 +1758,76 @@ components:
         - $ref: '#/components/schemas/OAuthClientCredentialsRequest'
         - $ref: '#/components/schemas/OAuthTokenExchangeRequest'
 
+    ScanReportRequest:
+      type: object
+      required:
+        - scan-report
+      properties:
+        scan-report:
+          $ref: '#/components/schemas/ScanReport'
+
+    ScanReport:
+      type: object
+      required:
+        - table-name
+        - snapshot-id
+        - filter
+        - projection
+        - metrics
+      properties:
+        table-name:
+         type: string
+        snapshot-id:
+          type: integer
+          format: int64
+        filter:
+          $ref: '#/components/schemas/Expression'
+        projection:
+          $ref: '#/components/schemas/Schema'
+        metrics:
+          type: object
```

Comment:
I think we need to be more specific here. This is an object where each property is a metric name and the value is a metric. We should probably have a `Metric` type as well that is used to describe the metrics objects.

Reply:
makes sense, added this